packages/scss/src/_typo.scss

```diff
@@ -7,39 +7,39 @@ html {
 }
 
 body {
-	background: $greyBackground;
-	color: $grey;
+	background: #F5F5F5;
+	color: _get($luTheme, "palette.colors.greyStandard");
 	font-family: "Source Sans Pro", Tahoma, sans-serif;
-	font-size: $font-standard;
-	line-height: $font-line-standard;
+	font-size: _get($luTheme, "font.size.standard");
+	line-height: _get($luTheme, "font.line.standard");
 	vertical-align: middle;
 }
 
 h1 {
-	font-size: $font-biggest;
+	font-size: _get($luTheme, "font.size.biggest");
 	font-weight: 600;
-	line-height: $font-line-biggest;
+	line-height: _get($luTheme, "font.line.biggest");
 	margin: 0 0 1.7rem;
```

Comment:
I'm not sure to understand the problem. 
Lucca Front set the body to 16px so all components are relative to this value.
If someone decide to change the body's font value, each components font will also change.
So, lucca banner fonts are also relative to the body font. If we want to reduce or increase the banner font size we will be able to set the value inside a "banner" scss/src/components file.